faq.md

```diff
@@ -0,0 +1,41 @@
+---
+layout: page
+title: FAQ
+permalink: /faq/
+---
+
+**Affiliations**
+- _Are you a part of the Drupal Association?_
+      No. We're a collective of volunteers.
+
+- _Didn't you have space in the Drupal Association's booth at DrupalCon Baltimore?_
+Yep. They donated it to us.
+
+- _Why?_
+Honestly, we didn't ask. We don't have the money to buy a booth space, so we took the gift we were offered.
+
+- _Did Dries have anything to do with it?_
+Nope.
+
+**Funding**
+
+We are a collective of volunteers that neither handles money nor has a way to receive or disburse funds.  
+
+- _Who bought the booth space?_
+No one. The Drupal Association donated it.
+
+- _Who paid for the stickers?_
+Digital Echidna, and they asked to do so anonymously.
```

Comment:
If Digital Echida asked to be anonymous why are you mentioning them now. I suggest either dropping that part of the sentence, or explain further that they've now gone public and given permission to mention them.